content/docs/started/getting-started-gke.md

```diff
@@ -56,7 +69,7 @@ Create an OAuth client ID to be used to identify Cloud IAP when requesting acces
         ```
         * `<name>` and `<project>` must have the same values as set in the next
           step when you run the deployment script.
-        * The deployment uses "kubeflow" by default for `<name>` but you can
+        * The deployment uses "ks-kubeflow" by default for `<name>` but you can
```

Comment:
Is that true? I believe the default is "kubeflow"

Reply:
Let me correct this. The correct answer is that it defaults to the `KFAPP` value.

Reply:
I've clarified the relationship between `<name>`, `KFAPP`, and `DEPLOYMENT_NAME`. Namely that when `kftcl.sh init $KFAPP ...` is run, `DEPLOYMENT_NAME` is assigned to the `KFAPP` value, saved in `${KFAPP}/env.sh` and subsequently used as the `<name>` value for the envoy-ingress endpoint.